src/graph.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+# -*- coding: utf -8 -*-
+
+from __future__ import unicode_literals, division
+
+
+class Graph(object):
+    """Defining class Graph."""
+    def __init__(self, iterable=None):
+        """Initiate an instance of class Graph."""
+        self._dict = {}
+        try:
+            for item in iterable:
+                self._dict.setdefault(str(item), [])
+        except TypeError:
+            if iterable is not None:
+                self._dict.setdefault(str(iterable), [])
+
+    def add_node(self, n):
+        """Add a node to graph"""
+        return self._dict.setdefault(str(n), [])
```

Comment:
What led you to decide to string-ify every incoming node value, instead of just setting the key into your dictionary as it was passed?